### ISSUE TITLE
Draw boundaries on the play area

### ISSUE THREAD

**Issue by BLFiedler** [phetsims/quadrilateral#49]
We have identified some issues with interaction/focus when the cursor/touch area leave the play area boundaries. These feel like they happen more unexpectedly as there is space in the sim that is not part of the play area, but does not look different than the play area (and is distinct form the dev bounds). 

Similar to RatioandProportions "bounding boxes", let's define the area the vertices can be moved visually.

Let's begin with just a simple line that contrasts well with the current blue background but is not the same color as the vertices/drag area. 

Up for future discussion will be whether the play area background differs from the rest of the background.

**Comment by BLFiedler** [comment]
The background colors will likely change, but for now it is pretty dark, so let's just use a white or light gray with perhaps a ~2-pixel width (just something with a bit of weight but not substantive for now). Something like this?

![image](https://user-images.githubusercontent.com/47331395/150400816-7d9bab02-d166-4ed9-b6ab-a09982f286a7.png)

**Comment by jessegreenberg** [comment]
Sounds good, added in the above commit. @BLFiedler can you please review?

**Comment by BLFiedler** [comment]
Great! I'll leave this open in case others have thoughts, but no more work to be done here for now.

**Comment by BLFiedler** [comment]
Related to the existence of the boundary - can we center the initial square to the play area boundary rather than the dev bounds?

**Comment by jessegreenberg** [comment]
Sounds good - I think the best way to do this is to fix the modelViewTransform so that model bounds are actually within the play area boundaries instead of the layout bounds. Currently model bounds are not set until layout, so we would need to have a mutable modelViewTransform. But setting bounds in `layout` is not necessary after work in https://github.com/phetsims/quadrilateral/issues/15. I am going to try
1) set modelBoundsProperty in the construcotor.
2) Remove `QuadrilateralNode.layout`, it is not necessary after #15.
3) Change the modelViewTransform to use model bounds in the constructor instead of layoutBounds.
4) Consider if modelBoundsProperty needs to be observable anymore, it shouldn't change currently.

**Comment by jessegreenberg** [comment]
OK this is done. Whether or not modelBoundsProperty should be observable will be decided in #55?

@BLFiedler can you please review?

**Comment by BLFiedler** [comment]
Great! Changes to the visuals on this will occur in a new visual mockup issue and behavior in #55.
